server/routerlicious/packages/routerlicious/src/kafka-service/command.ts

```diff
@@ -25,7 +25,7 @@ export function execute(
         .action((name: string, lambda: string) => {
             configureLogging(configOrPath);
             const lumberjackEngine = new WinstonLumberjackEngine();
-            Lumberjack.setup([lumberjackEngine], new LambdaSchemaValidator());
```

Comment:
The schema validator for lambdas was not applicable anymore?

Reply:
Had a conversation with Henrique offline. We decided to leave this as it is and think through how the schema validator can be made more generic.